src/java/picard/vcf/GenotypeConcordanceCounts.java

```diff
@@ -74,6 +86,72 @@ public void validateCountsAgainstScheme(final GenotypeConcordanceScheme scheme)
     }
 
     /**
+     * Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made
+     * If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.
+     */
+    public double calculateGenotypeConcordance(final GenotypeConcordanceScheme scheme, final boolean missingSitesFlag) {
+        double numerator = 0.0;
+        double denominator = 0.0;
+
+        scheme.validateScheme();
+
+        final TruthState[] allTruthStates = TruthState.values();
+        final CallState[] allCallStates = CallState.values();
+
+        for (final TruthState truthState : allTruthStates) {
+            for (final CallState callState : allCallStates) {
+                if(!missingSitesFlag && isMissing(truthState, callState)) {
+                    continue;
+                }
+                else {
+                    final TruthAndCallStates truthAndCallStates = new TruthAndCallStates(truthState, callState);
+                    final int count = getCount(truthAndCallStates);
+                    if (truthState.toString().equals(callState.toString())) {//we are on the diagonal
```

Comment:
The enum has a .equals() method but intelliJ doesn't like 
```truthState.equals(callState)```
saying they aren't comparable.

But if I do this: 
```truthState.name().equals(callState.name())```

it's fine with it. I think this would still be a string comparison though...